layouts/partials/homepage/encounters/card.html

```diff
@@ -0,0 +1,8 @@
+<div class="flex-1 flex-col w-1/4 mr-4 bg-border-dark-color">
+  <a class="block" href="{{ .link_to }}">
+    <img src="{{ .image_link }}" alt="" />
+    <div class="bg-border-dark-color">
+      <h2 class="text-2xl uppercase p-4 font-bold">{{ .name }}</h2>
+    </div>
+  </a>
+</div>
```

Comment:
I feel like some of this conflicts with the other active PR right now – you might want to discuss how to handle this with them. They technically submitted their PR first so I'm going to give them first dibs at merging, which might mess up your flow with card partials.

Reply:
@OHaiDanny same as in my other comment: If you want to I can check this partial once my PR is merged.